lib/taste_tester/tunnel.rb

```diff
@@ -44,15 +44,13 @@ def run
     end
 
     def cmd
-      if TasteTester::Config.user != 'root'
-        pid = '$$'
-      else
-        pid = '\\$\\$'
```

Comment:
OK so I decided to take a look at this deeper.

This code doesn't seem to work for me... but *also* the existing code does not  work. Unmodified code with tt ssh'ing as root:

```
# cat test_timestamp
#
```

Damn. What about non-root?

```
# cat test_timestamp
#
```

Nope, same thing. what the... when did this break?

What about with @flz 's changes? Nope, still don't get a PID in that file. Hmmm, do the commands work?

```
$ ssh root@host 'ps -o pgid= -p $(ps -o ppid= -p $$)'
 457791
```

Yeah, that seems to work. Hmmm, what about with the file redirection?

```
$ ssh root@host 'ps -o pgid= -p $(ps -o ppid= -p $$) > /etc/chef/test_timestamp'
```

```
# cat test_timestamp
 458433
```

Hmm, so the commands seem roughly correct, but this doesn't actually work with or without root...